db/schema.rb

```diff
@@ -10,11 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_180_331_030_345) do
+ActiveRecord::Schema.define(version: 20180322024915) do
 
   create_table "quotes", force: :cascade do |t|
     t.integer "age"
-    t.integer "trip_length"
+    t.decimal "trip_length"
```

Comment:
I introduced a migration to change this decimal to an integer, then changed my mind. I discarded the migration but accidentally committed the generated schema with an integer.